deps.bzl

```diff
@@ -149,8 +157,11 @@ def daml_deps():
     if "bazel_gazelle" not in native.existing_rules():
         http_archive(
             name = "bazel_gazelle",
-            urls = ["https://github.com/bazelbuild/bazel-gazelle/releases/download/0.17.0/bazel-gazelle-0.17.0.tar.gz"],
-            sha256 = "3c681998538231a2d24d0c07ed5a7658cb72bfb5fd4bf9911157c0e9ac6a2687",
+            urls = [
+                "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/bazel-gazelle/releases/download/v0.19.1/bazel-gazelle-v0.19.1.tar.gz",
+                "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.19.1/bazel-gazelle-v0.19.1.tar.gz",
+            ],
+            sha256 = "86c6d481b3f7aedc1d60c1c211c6f76da282ae197c3b3160f54bd3a8f847896f",
```

Comment:
This upgrade is not strictly necessary; I was trying to build `grpc-health-probe` from source and I was getting errors because the versions of `bazel_go` and `bazel_gazelle` were in conflict; specifically, the `go_repository` rule was changed in a breaking fashion and our version of the Gazelle plugin didn't know about the change.

Because I needed specifically a Linux binary to bundle into the Sandbox container, I ended up just using the pre-compiled version from its releases (which I wish I'd thought of sooner).

I left it in because it fixes an issue. 🤷‍♀